index.html

```diff
@@ -10,6 +10,7 @@
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 
 	</head>
 	<body>
```

Comment:
I'm unsure of the comment regarding having 2 body tags. I didn't add anything to the HTML document, aside from the jquery.min.js and the .js files. I definitely could have overlooked it, but i didn't see 2 either. HALP!!